tests/main/auto-refresh/task.yaml

```diff
@@ -10,29 +10,32 @@ execute: |
     echo "Install a snap from stable"
     snap install test-snapd-tools
     snap list | MATCH 'test-snapd-tools +[0-9]+\.[0-9]+'
+    snap set core refresh.schedule="0:00-$(date +%H:%M:%S --date=30seconds)"
```

Comment:
We still take seconds currently, they are used in the unit tests still. But I fixed the test, now that we have the 14day max interval the test does not need to set this schedule, it can use a different oen like 00:00-23:59.